metaflow/plugins/aip/kfp_cli.py

```diff
@@ -0,0 +1,18 @@
+from metaflow._vendor import click
+from metaflow.plugins.aip.aip_decorator import AIPException
+
+
+@click.group()
+def cli():
+    pass
+
+
+@cli.group(name="kfp", help="Deprecated, please use aip instead.")
+@click.pass_obj
+def kubeflow_pipelines(obj):
+    pass
+
+
+@kubeflow_pipelines.command(help="Deprecated, please use aip instead.")
+def run():
+    raise AIPException('Deprecated, please use "aip run" instead.')
```

Comment:
Nice! Just to confirm is this the only `python flow.py kfp *` CLI action we support? I think so but just wanted to triple check :) 

Stretch goal: Maybe expand the error message to be something like `python myflow.py aip run` or to capture the flow name and output it exactly might be the icing on the cake so they can copy paste exactly what they had in old versus new? (likely w/o arguments but the top level might be nice)